pkg/predicate/predicate.go

```diff
@@ -16,7 +16,12 @@ limitations under the License.
 
 package predicate
 
-import "sigs.k8s.io/controller-runtime/pkg/event"
+import (
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+var predicateLog = logf.KBLog.WithName("predicate").WithName("eventFilters")
```

Comment:
predicateLog --> log ?

Reply:
Don't we use unified `runtime log` ?

Reply:
I was suggesting to rename `predicateLog` --> `log`